demo/directives.html

```diff
@@ -459,5 +461,105 @@ <h4 class="lui underline">Demo</h4>
 			</p>
 			</div>
 		</article>
+        
+        <article id="luid-day-block">
+            <h2 class="lui divider below">luid-day-block</h2>
+			<p>
+				An element directive to to draw a day block calendar.
+			</p>
+			<div class="lui dashed divider">
+				<h4 class="lui underline">Supported arguments</h4>
+			</div>
+			<p>
+				<ul class="lui unstyled">
+					<li><code>show-day</code> : this argument must be a boolean, if true the directive shows the name of the day on top.</li>
+                    <li><code>first-color</code> : this color is used to fill the day number background. </li>
+                    <li><code>second-color</code> : this color is used to fill the month and year background.</li>
+				    <li><code>date</code> : this argument must be a moment. </li>
```

Comment:
it does not need to be a moment. moment just has to be able to parse it without specifying a format. if it can then `luifMoment` will work